Topic3/src/org/maugiam/tdd/main/WordWrapper.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * 
+ */
+package org.maugiam.tdd.main;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+/**
+ * @author Mauro J Giamberardino
+ *
+ */
+public class WordWrapper {
+
+	public static List<String> wrap(String string, Integer rowLength) {
+		ArrayList<String> res = new ArrayList<String>();
+		StringTokenizer tok = new StringTokenizer(string, " ");

```

Comment:
Warning: StringTokenizer is not that used nowadays. Favour the String.split () usage